app/error.tsx

```
@@ -0,0 +1,22 @@
+}
+
+export default function Error({ error, reset }: ErrorProps) {
+	return (
+		<main className="flex flex-col items-center justify-center gap-6">
+			<h1 className="text-3xl font-semibold">Something went wrong!</h1>
+			<p className="text-lg">{error.message}</p>
+
+			<button
+				className="inline-block bg-accent-500 px-6 py-3 text-lg text-primary-800"
+				onClick={reset}
+			>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)